content/articles/integrating-facebook-ads-into-your-android-application-in-android-studio/index.md

```diff
@@ -0,0 +1,172 @@
+In this case, we are going to add Banner Ads as follows;
+
+On the XML file(activity.xml), add a layout for the Banner Ads and implement it in the java file.This code below does everything for you.
+```kotlin
+<?xml version="1.0" encoding="utf-8">;
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+xmlns:App="http://schemas.android.com/apk/res-auto"
+xmlns:tools="http://schemas.android.com/tools"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+tools:context=".MainActivity">
+ 
+<LinearLayout
+android:id="@+id/banner_container"
+android:layout_width="match_parent"
+android:layout_height="wrap_content"
+android:layout_alignParentBottom="true"
+android:orientation="vertical"
+App:layout_constraintBottom_toBottomOf="parent"/>
+ 
+</RelativeLayout>
+
+```
+
+On your java activity, implement the Banner Ad by instantiating an Adview object. to do that, place this code in the `onCreate()` function.
+
+```kotlin
+adView = new AdView(this, "IMG_16_9_App_INSTALL#2029572424039676_2029575434039375", AdSize.BANNER_HEIGHT_60);
+ 
+// Find the AdContainer
+LinearLayout adContainer = (LinearLayout) findViewById(R.id.banner_container);
+ 
+adContainer.addView(adView);
+ 
+// Load an ad
+adView.loadAd();
+
+```
+In your activity's `onDestroy()` function, paste the following code. This is done to free up AdView's resources.
+
+```kotlin
+@Override
+protected void onDestroy() {
+if (adView != null) {
+adView.destroy();
+}
+super.onDestroy();
+}
+
+```
+
+With all that done, run your code and observe the output.
+For successful integration, the following output is obtained.
+
+![](engineering-education/integrating-facebook-ads-with-your-android-application-in-android-studio
+/output.png)
+
+You can alter the code to and other types of Ads like interstitial Ads and Native Ads in the same manner.
+
+### Why integrate Facebook ads?
+Consumers prefer in-App advertising over display advertising on mobile websites. Smartphone consumers find ads on the mobile web to be compacted and unpleasant. In-App advertisements are resized to fit the screen and seem better, increasing overall engagement.
+
+When establishing in-App advertisements, **accessibility** is another important issue to consider. In-App mobile advertising is easier to reach because most consumers have their phones with them all the time, wherever they go. You have access to a captive audience or users who are already using the App and are interested in what you have to say. It's easier to convert them because they're already interested.
+
+### Demerits of integrating into some Applications
+In-App advertisement as seen earlier has many advantages. Besides, it is not always Appealing to some users when ads keep popping in all the time.
+
+Here are some demerits in an in-App advertisement;
+
+- People/users might hate the App due to **disturbances from ads pop-up**. This may discourage users from using known Applications with in-App advertisements.

```

Comment:
The bold sentences are supposed to be at the beginning of the points as headings and not inline sentences.